File: help/pt_pt/exercicios_da_ficha_4/Ficha04_003/Ficha04_003.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha04_003 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Nota\t: " );
        int nota = Integer.valueOf(keyb.nextLine());
        if(nota >= 10 ){
            System.out.print("Aprovado" );
        }else{
            System.out.print("Reprovado" );
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:31

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_003/Ficha04_003.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Nota    : " ;
    $nota = intval(isset($_POST["nota"])? $_POST["nota"] : isset($_GET["nota"]) ? $_GET["nota"] : 0);
    
    echo $nota . "<br/>";// print input 
    if($nota >= 10 ){
        echo  "Aprovado" ;
    }else{
        echo  "Reprovado" ;
    }
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Mon, 2022 Sep 26 15:31

?>